src/posts/2021-01-20-ebpf-updates/index.md

```diff
@@ -0,0 +1,315 @@
+---
+path: "/news/ebpf-updates-2021-01"
+date: "2021-01-20T10:00:00.000Z"
+title: "eBPF Updates #3: Atomics Operations, Socket Options Retrieval, Syscall Tracing Benchmarks, eBPF in the Supply Chain"
+tags:
+  - _
+categories:
+  - Update
+---
+
+{{preview}}
+
+With the festive season, it would seem that eBPF blogging has cooled down a
+little. But the eBPF community is more active than ever, and we can be certain
+that 2021 will be full of new features, tutorials, deep dives, commercial news,
+and good surprises in general. In the meantime, here are all the latest news.
+Welcome to the third issue of the eBPF Updates, and Happy New Year!
+
+{{/preview}}
+
+## Foreword
+
+With the festive season, it would seem that eBPF blogging has cooled down a
+little. But the eBPF community is more active than ever, and we can be certain
+that 2021 will be full of new features, tutorials, deep dives, commercial news,
+and good surprises in general. In the meantime, here are all the latest news.
+Welcome to the third issue of the eBPF Updates, and Happy New Year!
+
+## New Resources
+
+### Introductory Resources
+
+<!--2020-11-25-->
+* [_BPF: The future of configs_](https://blog.habets.se/2020/11/BPF-the-future-of-configs.html),
+  from Thomas Habets.  
+  While many presentations focus on introducing eBPF's technical aspects, this
+  post takes a step back and describes why eBPF is “_such a big deal_”. For
+  network packet processing in particular, older frameworks (ipfwadm, ipchains,
+  iptables, nftables) are all about configuration, about feeding data for the
+  tables. By contrast, eBPF is about code and programming. This brings up
+  numerous possible use cases: packet routing, filesystem access control,
+  customization of TCP parameters, and more. This is a good read to help
+  understand what is at stake with eBPF.
+
+<!--2020-12-11-->
+* [_Linux Networking - eBPF, XDP, DPDK, VPP - What does all that mean?_](https://media.ccc.de/v/vnog-4-linux-networking-ebpf-xdp-dpdk-vp),
+  from Andree Toonk.  
+  Definitely oriented towards networking, this video introduces several of the
+  frameworks that have been used in the domain of fast networking over the last
+  years. The presentation is organized as a “journey”, that the presenter
+  experienced: From the quest for a fast traffic generator, that brought in
+  DPDK, to VPP and ultimately to eBPF and XDP, where you get fast packet
+  processing capabilities while keeping at hand all the features and goodness
+  from the Linux kernel. Andree also covers this topic
+  [on his blog](https://toonk.io/tag/networking/index.html).
+
+<!--2020-12-19-->
+* [_File Integrity Monitoring using eBPF_](https://www.devseccon.com/file-integrity-monitoring-using-ebpf-secadvent-day-19/),
+  from Sylvain Baubeau.  
+  After a brief introduction to eBPF, this post explains how it can improve
+  features such as _File Integrity Monitoring_ as implemented in the DataDog
```

Comment:
```suggestion
  features such as _File Integrity Monitoring_ as implemented in the Datadog
```

Reply:
The post I report, or the GitHub user name, say “DataDog” :shrug: But OK, let's stick to what their website says.